roles/sshd/defaults/main.yml

```diff
@@ -11,7 +11,7 @@ sshd_server_key_bits: "768"
 sshd_syslog_facility: "AUTH"
 sshd_log_level: "INFO"
 sshd_login_grace_time: "120"
-sshd_permit_root_login: "no"
+sshd_permit_root_login: "yes"
```

Comment:
Rather remove them to be consistent with the rest of our codebase since it's in our repo.